projects/steering/src/steering_can.c

```diff
@@ -33,6 +35,11 @@ StatusCode steering_can_process_event(Event *e) {
     case STEERING_CONTROL_STALK_EVENT_RIGHT_SIGNAL:
       CAN_TRANSMIT_LIGHTS(EE_LIGHT_TYPE_SIGNAL_RIGHT, (EELightState)e->data);
       break;
+    case STEERING_REGEN_BRAKE:

```

Comment:
Also, this should be `STEERING_REGEN_BRAKE_EVENT`, currently `STEERING_REGEN_BRAKE` is undefined which is causing the build to fail.